SR2021/2021-01-28-discord-mentoring.md

```diff
@@ -0,0 +1,29 @@
+---
+to: Teams who didn't attend session 1
+via: discord
+---
+
+Hey team $TLA,
+
+We noticed you didn't make it to the first league session. We wanted to reach
```

Comment:
Given the timing of this, we should probably change this to cover both.

Reply:
Yeah, at this point this framing is going to read rather oddly.

Reply:
02cf1a437f92ce1142fbe0a3823b237164754845